modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -166,7 +166,8 @@ public ObjectRelationship deleteObjectRelationship(
 				ObjectRelationshipConstants.TYPE_ONE_TO_MANY)) {
 
 			_objectFieldLocalService.deleteObjectField(
-				objectRelationship.getObjectFieldId2());
+				_objectFieldLocalService.getObjectField(
+					objectRelationship.getObjectFieldId2()));
```

Comment:
@marcelabc that way, we should rethink this solution. I have a suggestion. Let's talk later.